commands/lint.js

```diff
@@ -52,6 +67,52 @@ function api({strict = false, local = false, fix = false, eslintArgs = []} = {})
 	});
 }
 
+function tslintBin(context) {
+	try {
+		resolver.sync('tslint', {basedir: context});
+		return path.join(context, 'node_modules', '.bin', 'tslint');
+	} catch (e) {
+		return 'tslint';
+	}
+}
+
+function shouldTSLint(context) {
+	if (fs.existsSync(path.join(context, 'tslint.json'))) {
+		if (glob.sync('**/*.+(ts|tsx)', globOpts).length > 0) {
+			try {
+				return !spawn.sync(tslintBin(context), ['-v'], {stdio: 'ignore'}).error;
+			} catch (e) {
+				// ignore
```

Comment:
Should we throw a message here that tslint is not installed so there's no confusion about it?

Reply:
Most I'd prefer here would be a warning output, which I can add. Functionality-wise the feature is explicit about it's opt-in nature. If they haven't opted in, it doesn't make sense to outright error.

Reply:
I guess my thought was, if they use Typescript and they run `enact lint` and it just always returned success they may never know the problem was they didn't install tslint.